scene/mission-replay/src/main/resources/fxml/MissionView.fxml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Based on several examples from Oracle, I would name this file in lower-case, with underscore separator -
"mission_view.fxml", or just "mission.fxml". Android follows similar naming convention.